### ISSUE TITLE
Small issue: List showing nulls, not numbers, on render

### ISSUE THREAD

**Issue by james-pratama** [TypeCellOS/BlockNote#685]
On render, my lists are showing null instead of numbers. The numbers only show once I click the editor. 

<img width="798" alt="Screenshot 2024-04-03 at 8 17 11 AM" src="https://github.com/TypeCellOS/BlockNote/assets/94555269/0f7fea05-c5d6-44c7-995e-277efd8c7014">

Tried adding ref to BlockNoteView to auto focus editor but does not seem to work.

Is there a list somewhere of all the properties taken by BlockNoteView?

**Comment by YousefED** [comment]
@james-pratama could you provide steps or a sandbox to reproduce this?

**Comment by gbfgamemiya** [comment]
I have the same problem.

It can also be found at the following URL example.

https://www.blocknotejs.org/examples/basic/all-blocks

( Line of   Numbered List Item)

If you go back and forth between example > basic > (product line-up),
number list item becomes null when out of focus

![スクリーンショット 2024-04-12 065744](https://github.com/TypeCellOS/BlockNote/assets/68223965/3e4de14d-8d32-4b65-897d-8e2dc16eb729)

**Comment by rlancejohnson** [comment]
We also have this issue, and any help would be greatly appreciated.
https://blog.opengtm.ai/how-strategic-territory-planning-can-boost-sales-performance

<img width="739" alt="Screenshot 2024-05-03 at 12 05 01 AM" src="https://github.com/TypeCellOS/BlockNote/assets/36022919/f0e896f7-0ac8-4c88-8846-2f9524fb55d5">

**Comment by jinho-kim-osd** [comment]
We have a same issue as well.

**Comment by Dadangdut33** [comment]
I have the same issue, i fix it temporarily using this trick:
```tsx
useEffect(() => {
    let timeout: NodeJS.Timeout;
    if (editor) {
      timeout = setTimeout(() => {
        const firstBlock = editor.document[0];
        if (firstBlock) editor.updateBlock(firstBlock, { type: firstBlock.type });
      }, 100);
    }

    return () => {
      clearTimeout(timeout);
    };
  }, [editor]);
```

basically just update a block and it will not show as null after render

**Comment by kevinru2023** [comment]
I've encountered something similar as well, will try implementing temporary fix used by @Dadangdut33. 
<img width="987" alt="Screenshot 2024-06-02 at 9 46 46 AM" src="https://github.com/TypeCellOS/BlockNote/assets/105080383/46ebefc9-fab9-4ca2-a3e8-3baf2da0f449">

**Comment by matthewlipski** [comment]
Hi, haven't seen this issue before - if anyone can post steps to reproduce it that would be great!

**Comment by lamtuanamc** [comment]
@matthewlipski If you have multiple tabs using BlockNoteView, the numbered list will become null when you switch tabs.

Uploading Screen Recording 2024-06-13 at 09.13.13.mov…

**Comment by lamtuanamc** [comment]
@matthewlipski If you have multiple tabs using BlockNoteView, the numbered list will become null when you switch tabs. When status BlockNoteView is non-editable, when you reload page, the numbered list will become null too.


https://github.com/TypeCellOS/BlockNote/assets/37234239/2729b620-cc0c-494b-b70c-8f0b1599c640

**Comment by Dadangdut33** [comment]
> if anyone can post steps to reproduce it that would be great!

Hope it helps, so basically:
1. Intialize blocknote with data
2. Render blocknote
3. null will be shown for data that is using list until you click on the editor (or do an edit programmatically)

**Comment by matthewlipski** [comment]
Thanks! Helps a lot, we'll look into it next time we do a bugfix release

**Comment by danlupascu** [comment]
I'm having the same issue, but with the generated HTML. All items will show up as null once the ht is generating the HTML.

**Comment by tolypash** [comment]
I have the same issue when isEditable is false, on reload of page. When I click once, anywhere on the page it sets the numbers.

**Comment by YousefED** [comment]
closed by #1113 🥳
